module/zcommon/zfs_prop.c

```diff
@@ -245,6 +245,14 @@ zfs_prop_init(void)
 		{ NULL }
 	};
 
+	static zprop_index_t volmode_table[] = {
+		{ "default",	ZFS_VOLMODE_DEFAULT },
+		{ "geom",	ZFS_VOLMODE_GEOM },
```

Comment:
We were thinking to rename `geom` to `iokit` for our platform, but I can get behind `full` as more platform generic.